penguin/engine.py

```
@@ -2327,6 +2356,64 @@ async def _handle_responses_tool_call(
+            )
+            session_id = str(getattr(session, "id", "") or "unknown").strip()
+            workspace_path = getattr(cm, "workspace_path", None)
+            if workspace_path is None:
+                return None
+            return (
+                Path(workspace_path)
+                / "conversations"
+                / "tool-results"
+                / session_id
+            )
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)